starter-core/src/main/java/io/micronaut/starter/feature/dev/ControlPanel.java

```diff
@@ -0,0 +1,77 @@
+package io.micronaut.starter.feature.dev;
+
+import io.micronaut.context.env.Environment;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+import io.micronaut.starter.feature.config.ApplicationConfiguration;
+import io.micronaut.starter.feature.other.Management;
+import jakarta.inject.Singleton;
+
+/**
+ * Micronaut Control Panel feature.
+ *
+ * @author Álvaro Sánchez-Mariscal
+ * @since 4.0.0
+ */
+@Singleton
+public class ControlPanel implements Feature {
+
+    public static final String NAME = "control-panel";
+
+    private static final Dependency.Builder CONTROL_PANEL_DEPENDENCY = Dependency.builder()
+            .groupId("io.micronaut.controlpanel")
```

Comment:
Consider creating a method `MicronautDependencyUtils.controlpanel()`instead of `Dependency.builder()
            .groupId("io.micronaut.controlpanel")`